chapter_01/p01_is_unique.py

```diff
@@ -38,7 +38,7 @@ def is_unique_bit_vector(string):
         checker |= 1 << val
     return True
 
-def allUniqueChar(s:str):
+def alluniquechar(s: str) -> bool:
```

Comment:
`all_unique_char` would be more consistent with the names of the other functions in this module.

Also, it would be nice to indicate in the name what makes this implementation different from the others.